<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:abcast/1 2024-10-30 13:57:13.176 ***</b>
=CRASH REPORT==== 30-Oct-2024::13:57:13.176343 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: &lt;0.244475.0&gt;
    registered_name: my_test_name
    exception exit: {&lt;0.244473.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [&lt;0.244473.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3453
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-30 13:57:15.274 ***</b>
=ERROR REPORT==== 30-Oct-2024::13:57:15.274298 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244502.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1973}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1934}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-10-30 13:57:15.275 ***</b>
=CRASH REPORT==== 30-Oct-2024::13:57:15.274769 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244509.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2426)
    ancestors: [<0.244502.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244502.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-30 13:57:55.131 ***</b>
=ERROR REPORT==== 30-Oct-2024::13:57:55.131737 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248851.0&gt;,
                               {#Ref&lt;0.3306660998.1786249217.131500&gt;,1007067,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.3306660998.1786249217.131500&gt;,1007067,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-10-30 13:57:55.132 ***</b>
=CRASH REPORT==== 30-Oct-2024::13:57:55.132153 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248872.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3306660998.1786249217.131500&gt;,1007067,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [&lt;0.248851.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4505
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init_local_registered_parent/1 2024-10-30 13:57:55.155 ***</b>
=CRASH REPORT==== 30-Oct-2024::13:57:55.154407 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_local/2
    pid: &lt;0.248877.0&gt;
    registered_name: my_server
    exception exit: {&lt;0.248875.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [foobar]
    message_queue_len: 0
    messages: []
    links: [&lt;0.248875.0&gt;]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3439
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init_global_registered_parent/1 2024-10-30 13:57:55.178 ***</b>
=CRASH REPORT==== 30-Oct-2024::13:57:55.178097 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global/2
    pid: &lt;0.248882.0&gt;
    registered_name: []
    exception exit: {&lt;0.248880.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [&lt;0.248880.0&gt;]
    message_queue_len: 0
    messages: []
    links: [&lt;0.248880.0&gt;]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3420
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-10-30 14:02:33.345 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:02:33.345256 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254668.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.3306660998.1787559937.195765&gt;,6848,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1694)
    ancestors: [&lt;0.254666.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28125
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-30 14:02:43.006 ***</b>
=ERROR REPORT==== 30-Oct-2024::14:02:43.006132 ===
** Generic server <0.255951.0> terminating 
** Last message in was {nodedown,'peer_down_crash-417828-11742@a163d9b25382'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-417828-11742",
                              peer_down => crash},
                            'peer_down_crash-417828-11742@a163d9b25382',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,49,46,50,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-417828-11742",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQGExNjNkOWIyNTM4MgAD588AAAAAZyIq3g=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-417828-11742@a163d9b25382'}},
                            {<0.255933.0>,
                             #Ref<0.3306660998.1779957763.199511>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-417828-11742@a163d9b25382'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-10-30 14:02:43.007 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:02:43.006828 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255951.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-417828-11742@a163d9b25382'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2476)
    ancestors: [<0.255933.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11329
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-30 14:02:43.195 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:02:43.195115 ===
'global' at node test_server@a163d9b25382 requested disconnect from node 'dist_localhost-417860-11742@localhost' in order to prevent overlapping partitions

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-30 14:02:43.930 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:02:42.928427 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-417956-11742@a163d9b25382' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-30 14:02:44.423 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:02:44.423174 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-418020-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-10-30 14:02:44.498 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:02:44.498246 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-418212-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-30 14:03:00.183 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:02:59.181632 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-406821-11742@a163d9b25382' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-30 14:03:00.611 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:03:00.611237 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-418884-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-10-30 14:03:00.615 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:03:00.615019 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-406853-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:02.886 ***</b>
=ERROR REPORT==== 30-Oct-2024::14:03:02.886322 ===
** Generic server <0.256302.0> terminating 
** Last message in was {'EXIT',<0.256299.0>,
                               {#Ref<0.3306660998.1779957763.204165>,1614388,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-419620-11742",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-419620-11742@a163d9b25382',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,49,46,50,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-419620-11742",
                                     "-user","peer"],
                                    #Port<0.3120>,undefined,<<>>,running,
                                    {<0.256299.0>,
                                     #Ref<0.3306660998.1779957763.204227>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3306660998.1779957763.204165>,1614388,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:02.887 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:03:02.886977 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256302.0>
    registered_name: []
    exception exit: {#Ref<0.3306660998.1779957763.204165>,1614388,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256299.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3120>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 13746
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:03.451 ***</b>
=ERROR REPORT==== 30-Oct-2024::14:03:03.451317 ===
** Generic server <0.256339.0> terminating 
** Last message in was {'EXIT',<0.256300.0>,
                           {#Ref<0.3306660998.1780219908.15502>,2178197,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "IWKUYKUGRUMTKTGTHNDO","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a163d9b25382.2024-10-30_13.47.25/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-407077-11742",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a163d9b25382.2024-10-30_13.47.25/make_test_dir.stdlib_test.logs/run.2024-10-30_13.48.01/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-407077-11742",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.4/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-407077-11742@a163d9b25382',
                                    "/usr/local/lib/erlang-26.2.5.4/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-407077-11742",
                                     "-setcookie","IWKUYKUGRUMTKTGTHNDO",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a163d9b25382.2024-10-30_13.47.25/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-407077-11742",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@a163d9b25382.2024-10-30_13.47.25/make_test_dir.stdlib_test.logs/run.2024-10-30_13.48.01/log_private/",
                                     "-user","peer"],
                                    #Port<0.3147>,undefined,<<>>,running,
                                    {<0.256300.0>,
                                     #Ref<0.3306660998.1780219908.17017>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.3306660998.1780219908.15502>,2178197,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:03.452 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:03:03.452097 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256339.0>
    registered_name: []
    exception exit: {#Ref<0.3306660998.1780219908.15502>,2178197,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.256300.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3147>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 10958
    stack_size: 29
    reductions: 20926
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:03.854 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:03:02.854155 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-512994-11742@a163d9b25382' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:03.987 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:03:03.987219 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-407045-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:04.159 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:03:04.159244 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-512930-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-10-30 14:03:04.317 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:03:04.317265 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-419780-11742@a163d9b25382'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-30 14:24:43.635 ***</b>
=ERROR REPORT==== 30-Oct-2024::14:24:43.635656 ===
** Generic server <0.333228.0> terminating 
** Last message in was {'EXIT',<0.333225.0>,
                               {#Ref<0.3306660998.1816920065.71408>,351236,
                                [{child1,<0.333234.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.333228.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.333234.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458515],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3306660998.1816920065.71408>,351236,
    [{child1,<0.333234.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-30 14:24:43.636 ***</b>
=ERROR REPORT==== 30-Oct-2024::14:24:43.635655 ===
** Generic server <0.333227.0> terminating 
** Last message in was {'EXIT',<0.333225.0>,
                               {#Ref<0.3306660998.1816920065.71408>,351236,
                                [{child1,<0.333234.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.333227.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.333233.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458515],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.3306660998.1816920065.71408>,351236,
    [{child1,<0.333234.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-30 14:24:43.636 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:24:43.636115 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.333228.0>
    registered_name: []
    exception exit: {#Ref<0.3306660998.1816920065.71408>,351236,
                     [{child1,<0.333234.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.333225.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14895
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-10-30 14:24:43.636 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:24:43.636376 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.333227.0>
    registered_name: []
    exception exit: {#Ref<0.3306660998.1816920065.71408>,351236,
                     [{child1,<0.333234.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [<0.333225.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 14981
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-10-30 14:25:27.125 ***</b>
=CRASH REPORT==== 30-Oct-2024::14:25:27.125588 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.433385.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2299)
    ancestors: [bridge9212,<0.433380.0>]
    message_queue_len: 0
    messages: []
    links: [<0.433386.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4058
  neighbours:
    neighbour:
      pid: <0.433386.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.433385.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2024-10-30 14:51:50.105 ***</b>
=WARNING REPORT==== 30-Oct-2024::14:51:50.101466 ===
Process <0.436865.0> on node 'test_server@a163d9b25382' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Wed Oct 30 2024 13:48:01<!--/date--><br />
</div>
</center>
</body>
</html>
